src/index.html

```diff
@@ -11,9 +11,13 @@
     <div id="main"></div>
     <script type="text/javascript" src="main.js"></script>
     <script type="text/javascript">
-        Elm.Main.init({
+        const app = Elm.Main.init({
             node: document.getElementById('main'),
         })
+
+        app.ports.sendMessage.subscribe(function(message) {
+            navigator.clipboard.writeText(message)
+        });
```

Comment:
Might be worth a caveat that this API is [not currently supported in Safari](https://caniuse.com/#feat=mdn-api_clipboard_writetext)

Reply:
Oh interesting, looks like `document.execCommand('copy')` has better support but mdn says that it's not supported in Safari either 
<img width="677" alt="Screen Shot 2020-04-17 at 09 46 55" src="https://user-images.githubusercontent.com/11618797/79550645-8c8e3f80-8090-11ea-992b-02099382a2b9.png">